scim-sdk-server/src/test/java/de/captaingoldfish/scim/sdk/server/patch/PatchAddResourceHandlerTest.java

```diff
@@ -2364,81 +2364,6 @@ public void testMsAzureBehaviourForMultivaluedComplexTypesWithFilterInPathExpres
     Assertions.assertEquals("work", patchedUser.getEmails().get(1).getType().get());
   }
 
-  /**
-   * verifies that if the ms-azure patch filter workaround is active that the values of the filter are added if
-   * the filter does not match an existing entry.<br>
-   * <br>
-   * Example:<br>
-   * the request
-   *
-   * <pre>
-   * {
-   *   "schemas": [
-   *     "urn:ietf:params:scim:api:messages:2.0:PatchOp"
-   *   ],
-   *   "Operations": [
-   *     {
-   *       "op": "replace",
-   *       "path": "emails[type eq \"work\"].value",
-   *       "value": "max@mustermann.de"
-   *     }
-   *   ]
-   * }
-   * </pre>
-   *
-   * must result in
-   *
-   * <pre>
-   * {
-   *   "schemas" : [ "urn:ietf:params:scim:schemas:core:2.0:User" ],
-   *   "emails" : [
-   *     {
-   *       "type" : "work",
-   *       "value" : "max@mustermann.de"
-   *     }
-   *   ],
-   *   "meta" : {
-   *     "lastModified" : "2023-02-05T11:56:25.8049737+01:00"
-   *   }
-   * }
-   * </pre>
-   *
-   * if the user did have the email "erika@mustermann.de" before
-   */
-  @Test
-  public void testMsAzureBehaviourForMultivaluedComplexTypesWithFilterInPathExpression2Replace()
```

Comment:
These tests showed that the patch couldn't work to replace a single value in a list.

If you define emails as list:
```
    User user = User.builder().emails(Arrays.asList(
            Email.builder().primary(true).value("erika@mustermann.de").build(),
            Email.builder().primary(false).value("erika-secondary@mustermann.de").build()
    )).build();
```
This will fail:  `Assertions.assertEquals(2, patchedUser.getEmails().size(), patchedUser.toPrettyString());` - only one email remains there.